### PR TITLE
docs: add Foundry era test node tutorial

### PR DESCRIPTION
**Pull request by bxpana**
- add tutorial for using era_test_node with Foundry to call swap router contract for Uniswap

### CODE REVIEW

File: content/tutorials/foundry-era-test-node/10.index.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+::
+
+---
+
+## 1. Setting Up the Environment
+
+### Fork ZKsync Era Mainnet
+
+Start `era_test_node` and fork the ZKsync Era mainnet:
```
Comment (sarahschwartz) [inline content/tutorials/foundry-era-test-node/10.index.md:50]:
```suggestion
Start `era_test_node` and fork ZKsync Era mainnet:
```

File: content/tutorials/foundry-era-test-node/10.index.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+
+```bash
+era_test_node fork mainnet
+```
+
+For debug mode (provides more transaction details):
+
+```bash
+era_test_node -d fork mainnet
```
Comment (sarahschwartz) [inline content/tutorials/foundry-era-test-node/10.index.md:59]: This doesn't work for me, it says there is no `-d` flag. What version of `era_test_node` do you have?

File: content/tutorials/foundry-era-test-node/10.index.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+```
+
+---
+
+## 2. Streamlining with Environment Variables
+
+Create a `.env` file in your project directory to store environment variables:
```
Comment (sarahschwartz) [inline content/tutorials/foundry-era-test-node/10.index.md:66]: Should the user create a new project directory before this step?

File: content/tutorials/foundry-era-test-node/10.index.md [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+ETH_RPC_URL=http://localhost:8011
+PRIVATE_KEY=rich_wallet_private_key_here
+ETH_GAS_LIMIT=10000000    # Optional
+WETH=0x5AEa5775959fBC2557Cc8789bC1bf90A239D9a91
+MY_ADDRESS=your_wallet_address_here
+SWAP_ROUTER=0x99c56385daBCE3E81d8499d0b8d0257aBC07E8A3
+```
+
+- **ETH_RPC_URL**: The URL of your local `era_test_node` instance.
+- **PRIVATE_KEY**: One of the private keys from the `era_test_node` accounts.
```
Comment (sarahschwartz) [inline content/tutorials/foundry-era-test-node/10.index.md:78]: Can you link to where to find these?

File: content/tutorials/foundry-era-test-node/10.index.md [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+- **ETH_GAS_LIMIT**: Optional gas limit for transactions.
+- **WETH**: WETH token address on ZKsync Era.
+- **MY_ADDRESS**: Your wallet address.
```
Comment (sarahschwartz) [inline content/tutorials/foundry-era-test-node/10.index.md:81]: Should the address be the public key of the private key above?

File: content/tutorials/foundry-era-test-node/10.index.md [hunk 6]
```diff
@@ -0,0 +1,290 @@
...
+
+- **`<AMOUNT_TO_APPROVE>`**: The amount of WETH you plan to swap (in wei).
+
+---
+
+## 7. Selecting an ERC20 Token and Swap Pair
+
+### Find an ERC20 Token
+
+- Use the [ZKsync Era Tokens List](https://era.zksync.io/tokens) to choose an ERC20 token.
```
Comment (sarahschwartz) [inline content/tutorials/foundry-era-test-node/10.index.md:163]: This link redirects to the docs for me

**Review by sarahschwartz — COMMENTED** [review]
Great idea for a guide, well done! Just have some minor questions + nits

File: content/tutorials/foundry-era-test-node/_dir.yml [hunk 7]
```diff
@@ -0,0 +1,29 @@
+title: Interacting with Uniswap on a Fork of ZKsync Era Using `era_test_node` and Foundry
+featured: true
```
[the remark this answers is not in the document]
Comment (bxpana) [reply content/tutorials/foundry-era-test-node/_dir.yml:2]: I'll remove this one as featured. It's not as important as the others.

File: content/tutorials/foundry-era-test-node/10.index.md (hunk 2 above)
Comment (bxpana) [reply content/tutorials/foundry-era-test-node/10.index.md:59]: era_test_node 0.1.0-alpha.27

Tested and it worked for me.

Comment (bxpana) [reply content/tutorials/foundry-era-test-node/10.index.md:59]: Added a note about the version too

**Review by sarahschwartz — APPROVED** [review]

**Pull request merged**